sdk/python/feast/infra/offline_stores/remote.py

```diff
@@ -39,45 +40,24 @@ def __init__(
         entity_df: Union[pd.DataFrame, str],
         # TODO add missing parameters from the OfflineStore API
     ):
-        # Generate unique command identifier
-        self.command = str(uuid.uuid4())
         # Initialize the client connection
         self.client = fl.connect(
             f"grpc://{config.offline_store.host}:{config.offline_store.port}"
         )
-        # Put API parameters
-        self._put_parameters(feature_refs, entity_df)
+        self.feature_refs = feature_refs
+        self.entity_df = entity_df
 
-    def _put_parameters(self, feature_refs, entity_df):
-        historical_flight_descriptor = fl.FlightDescriptor.for_command(self.command)
+    # TODO add one specialized implementation for each OfflineStore API
+    # This can result in a dictionary of functions indexed by api (e.g., "get_historical_features")
+    def _put_parameters(self, command_descriptor, command):
```

Comment:
command is redundant now

Reply:
I was tempted to keep it for further extensions but in the spirit of keeping the code clean, I removed it, thanks!